patches/minecraft/net/minecraft/client/renderer/tileentity/RenderItemFrame.java.patch

```diff
@@ -1,15 +1,26 @@
 --- ../src-base/minecraft/net/minecraft/client/renderer/tileentity/RenderItemFrame.java
 +++ ../src-work/minecraft/net/minecraft/client/renderer/tileentity/RenderItemFrame.java
-@@ -106,6 +106,9 @@
+@@ -99,15 +99,18 @@
+             GlStateManager.func_179140_f();
+             int i = p_82402_1_.func_82333_j();
+ 
+-            if (item == Items.field_151098_aY)
++            if (item == Items.field_151098_aY || item instanceof net.minecraft.item.ItemMap)
+             {
```

Comment:
Why not JUST the instanceof check? Yes it's slower, but you loose all optimization anyways because all other items will run the instanceof check anyways.